app/services/transaction_search/searcher.rb

```diff
@@ -26,22 +26,20 @@ class Searcher
       ]
     end
 
-    # Shorthand method if you only want the default searchers
-    def self.search(order_details, params)
-      new.search(order_details, params)
+    # Searchers used on the billing page
+    def self.billing_searchers(include_facilities = false)
```

Comment:
Your changes aren't bad, but I don't love the change in API. What if we did:

```
def self.billing_search(order_details, params, facility:)
  searchers = default_searchers.dup
  searchers.unshift(TransactionSearch::FacilitySearcher) if facility.cross_facility?
  new(searchers).search(order_details, params)
end
```
Or use a `include_facility:` keyword to keep that logic out of this.